BlazarService/src/main/java/com/hubspot/blazar/util/ModuleBuildLauncher.java

```diff
@@ -84,7 +87,7 @@ private BuildConfig fetchBuildpack(GitInfo gitInfo) throws IOException, NonRetry
   }
 
   private static BuildConfig mergeConfig(BuildConfig primary, BuildConfig secondary) {
```

Comment:
if we have this `mergeConfig()` method, then all the `BuildConfig` fields should be `Optional<>` to make merging easier